lib/kernel/doc/src/gen_tcp.xml

```diff
@@ -258,6 +258,12 @@ do_recv(Sock, Bs) ->
 {tcp_error, Socket, Reason}</code>
         <p>The optional <c><anno>Timeout</anno></c> parameter specifies a
           time-out in milliseconds. Defaults to <c>infinity</c>.</p>
+        <note>
+          <p>Keep in mind that if the underlying OS <c>connect()</c> call returns
+            with a timeout, <c>gen_tcp:connect</c> will also return with a timeout
```

Comment:
Should this say "returns a timeout"?

Reply:
```suggestion
            a timeout, <c>gen_tcp:connect</c> will also return a timeout
```

Reply:
You're right. My bad for copy&pasting directly from someone else's email :)